v2/cli/init_templates.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for a non-git project
```

Comment:
This comment isn't accurate. This template is used for projects that involve git as well as the ones that don't.